src/pages/Content/index.js

```diff
@@ -10,10 +10,18 @@ chrome.runtime.onMessage.addListener(function (msgObj) {
       console.log('Should start recording here');
       sendMessage({ action: 'test', value: 'Hi from content (recorder)!' });
       break;
+    case 'stopRecording':
+      console.log('Should stop recording here');
+      sendMessage({ action: 'test', value: 'Hi from content (recorder)!' });
```

Comment:
The action and value are not consistent with the stoprecording event, no?

Reply:
and here we will need to call some kind of function to stop the recording and send back the Popup all the events we found.

Reply:
This PR is for updating the button click sendMessages, and the content script sendMessage is there to keep the same content of previous cases. I think we should update the sendMessage or respective actions for the Content script in the separate PR.